vpdq/python/vpdq.pyx

```diff
@@ -144,6 +144,8 @@ def computeHash(
 
     if downsample_height == 0:
         downsample_height = vid.get(cv2.CAP_PROP_FRAME_HEIGHT)
+    
+    vid.release()
```

Comment:
(more familiar with python): There's probably a nice way to do this with a with/finally, but this is fine for the purpose of this CLI.